docroot/modules/humsci/hs_paragraph_types/modules/hs_text_area/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "su_humsci_theme",
+  "description": "Drupal 8 subtheme of Stanford Basic for the School of Humanities and Sciences",
+  "author": "Stanford Web Services",
+  "version": "1.0.0",
+  "license": "MIT",
+  "repository": "https://github.com/SU-HSDO/suhumsci",
+  "devDependencies": {
+    "bourbon": "^5.0.0",
+    "bourbon-neat": "^1.8.0",
+    "decanter": "^3.0.0",
+    "grunt": "^1.0.3",
+    "grunt-autoprefixer": "^3.0.3",
```

Comment:
We need to update our standard file, this is out of date. grunt-autoprefixer is out and autoprefixer is in.